cortexlab/+git/update.m

```diff
@@ -17,14 +17,15 @@ function update(fatalOnError, scheduled)
 end
 disp('Updating code...')
 
-% Get the path to the Git exe
+% Get the path to the Git exe (can we assume this is unnecessary)? 
 gitexepath = getOr(dat.paths, 'gitExe');
 if isempty(gitexepath)
-  [~,gitexepath] = system('where git');
+  [~,gitexepath] = system('where git'); % this doesn't always work
 end
 gitexepath = ['"', strtrim(gitexepath), '"'];
 
-% Temporarily change directory into Rigbox
+% Temporarily change directory into Rigbox -> isn't it safe to assume we're
+% already in Rigbox? (since we're running this file, presumably we are?)
```

Comment:
It isn't safe to assume this.  All Rigbox functions are in the paths so there's no need to be in the Rigbox folder in order to run them.  The code has been carefully set up so that everything runs regardless of working directory.  This is important as users often want to run Rigbox in conjunction with local functions that aren't on the paths.  If you're happy with this you can remove this comment push

Reply:
Oh and the reason I cd here is because pulling from outside the git repo wasn't working for me.  Not sure why